python/ray/tests/test_basic_2.py

```diff
@@ -634,6 +637,32 @@ def test_get_correct_node_ip():
         assert found_ip == "10.0.0.111"
 
 
+def test_load_code_from_local():
+    code_test = """
```

Comment:
Can you add a comment to explain what cases this case is testing?

Reply:
Fixed. Thanks.

Reply:
Can you also add this?
```
This tests the following two cases when `load_code_from_local=True`:
1) Load nested class. 
2) Load a class defined in the `__main__` module.
```

Reply:
OK.